src/tls_stream.rs

```diff
@@ -56,25 +59,49 @@ impl Builder {
         self
     }
 
-    #[allow(dead_code)]
-    /* pub */ fn cert_store(&mut self, cert_store: CertStore) -> &mut Builder {
+    /// Specifies a custom certificate store which is later used when validating
+    /// a server's certificate.
+    ///
+    /// This option is only used for client connections and is used to construct
+    /// the certificate chain which the server's certificate is validated
+    /// against.
```

Comment:
We'll probably want to note here that the certificates in this store are not implicitly trusted - it certainly surprised me.